development.ini

```diff
@@ -68,7 +68,7 @@ db_schema = public
 # For virtuoso:
 # db_schema = assembl
 db_database = assembl
-# db_host = localhost
+db_host = database
```

Comment:
@swergas we'll have to see if we can devote time to this internally, I would be in favour.